### PR TITLE
LPS-191958 When you rename a Data Set entry, it's possible to leave the name field blank

### PR DESCRIPTION
**Pull request by thektan**
https://liferay.atlassian.net/browse/LPS-191958

**Rename Data Set Entry Modal:**
- Disables save when input is blank AND while saving
- Added an additional `!response.ok` check in case the response errors
- Renamed `handleOnBlur` -> `onBlur` for consistency. 

## Demo

https://github.com/liferay-frontend/liferay-portal/assets/4496722/94c97d85-9c1c-4194-9f59-661deeb39bf9

### CODE REVIEW

**Comment by thektan** [comment]
ci:test:relevant

**Review by markocikos — CHANGES_REQUESTED** [review]
@thektan Please see comments inline.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx [hunk 1]
```diff
@@ -790,7 +801,10 @@ const RenameFDSEntryModalContent = ({
 			<ClayModal.Footer
 				last={
 					<ClayButton.Group spaced>
-						<ClayButton onClick={saveFDSEntryRename}>
+						<ClayButton
+							disabled={!fdsEntryLabel || saveButtonDisabled}
```
[the remark this answers is not in the document]
Comment (markocikos) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx:805]: I just opened [LPS-203066 Ensure consistent behavior of disabling buttons during validation and save in DSM](https://liferay.atlassian.net/browse/LPS-203066)

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
@thektan LGTM! :sparkles:

**Comment by mateusralv** [comment]
Hi @thektan the failures are not related.

[Testray comparison](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1701772130759&testrayRunIdA=2468462451&testrayRunIdB=2469028576)

**Comment by thektan** [comment]
ci:forward:force

**Pull request closed without merging**